syntax = "proto3";

message LocationMessage {
    int32 person_id = 1;
    string latitude = 2;
    string longitude = 3;
    int64 timestamp = 4;
}

service LocationService {
    rpc Create(LocationMessage) returns (LocationMessage);
}